TESTS/spm/server/tests.c

```diff
@@ -341,26 +345,27 @@ PSA_TEST_SERVER(msg_read_truncation)
 
     signals = psa_wait_any(PSA_WAIT_BLOCK);
     if ((signals & TEST_MSK) == 0) {
-        test_status = ((test_status != PSA_SUCCESS) ? test_status : PSA_GENERIC_ERROR);
+        test_status = PSA_GENERIC_ERROR;
     }
 
     psa_get(TEST_MSK, &msg);
     if (msg.type != PSA_IPC_MSG_TYPE_CALL) {
         test_status = ((test_status != PSA_SUCCESS) ? test_status : PSA_GENERIC_ERROR);
     }
 
-    read_size = psa_read(msg.handle, 5, buff, sizeof(buff));
-    if ((6 != read_size) ||
-        (msg.size != 11) ||
+    read_size = psa_read(msg.handle, 1, buff, 11);
+    if ((msg.size[1] != read_size) ||
+        ((msg.size[0] + msg.size[1] + msg.size[2]) != 11) ||
         (buff[6] != 0)   ||
-        (strncmp(buff, "ghijkn", 6) != 0)) {
+        (strncmp(buff, "fghijk", 6) != 0)) {
```

Comment:
I'm curious. Was there any particular reason for the letter change?

Reply:
we've changed the buffers because the prevoius tests all sent the same vectors